tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -574,4 +586,84 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.GhQuantities",
                                     upper_bound);
   test_shift_deriv_functions_analytic(solution, grid_size, lower_bound,
                                       upper_bound);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(GeneralizedHarmonic::Tags::DerivSpatialMetricCompute<
+            3, Frame::Inertial>::name() == "deriv(SpatialMetric)");
+  CHECK(GeneralizedHarmonic::Tags::DerivLapseCompute<3,
+                                                     Frame::Inertial>::name() ==
+        "deriv(Lapse)");
+  CHECK(GeneralizedHarmonic::Tags::DerivShiftCompute<3,
+                                                     Frame::Inertial>::name() ==
+        "deriv(Shift)");
+
+  // Check that the compute items return the correct values
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> dist(-1., 1.);
+  std::uniform_real_distribution<> dist_positive(1., 2.);
+  const auto nn_generator = make_not_null(&generator);
+  const auto nn_dist = make_not_null(&dist);
+  const auto nn_dist_positive = make_not_null(&dist_positive);
+
+  const auto lapse = make_with_random_values<Scalar<DataVector>>(
+      nn_generator, nn_dist_positive, used_for_size);
+  const auto shift =
+      make_with_random_values<tnsr::I<DataVector, 3, Frame::Inertial>>(
+          nn_generator, nn_dist, used_for_size);
+  const auto spatial_metric = [&]() {
+    auto spatial_metric_l =
+        make_with_random_values<tnsr::ii<DataVector, 3, Frame::Inertial>>(
+            nn_generator, nn_dist, used_for_size);
+    // Make sure spatial_metric isn't singular by adding
+    // large enough positive diagonal values.
+    for (size_t i = 0; i < 3; ++i) {
+      spatial_metric_l.get(i, i) += 2.;
+    }
+    return spatial_metric_l;
+  }();
+  const auto expected_deriv_lapse =
+      make_with_random_values<tnsr::i<DataVector, 3, Frame::Inertial>>(
+          nn_generator, nn_dist_positive, used_for_size);
+  const auto expected_deriv_shift =
+      make_with_random_values<tnsr::iJ<DataVector, 3, Frame::Inertial>>(
+          nn_generator, nn_dist, used_for_size);
+  const auto expected_deriv_spatial_metric =
+      make_with_random_values<tnsr::ijj<DataVector, 3, Frame::Inertial>>(
+          nn_generator, nn_dist, used_for_size);
+
+  const auto spacetime_normal_vector =
+      gr::spacetime_normal_vector(lapse, shift);
+  const auto& inverse_spatial_metric =
+      determinant_and_inverse(spatial_metric).second;
+  const auto inverse_spacetime_metric =
+      gr::inverse_spacetime_metric(lapse, shift, inverse_spatial_metric);
+
+  const auto phi = GeneralizedHarmonic::phi(
+      lapse, expected_deriv_lapse, shift, expected_deriv_shift, spatial_metric,
+      expected_deriv_spatial_metric);
+
+  const auto other_box = db::create<
```

Comment:
`other_box` is a little bit of a confusing name if there's only the one in the test (which I think is true...).